nc2zarr/converter.py

```diff
@@ -58,6 +58,8 @@ class Converter:
            If there is no existing dataset, one will be created regardless
            of the value of this parameter.
     :param output_append_dim:
+    :param output_append_mode: behaviour to use when appending data which
```

Comment:
Which values are supported?

Reply:
See corresponding enum

Reply:
Now listed in docstring (commit cd5329b).